forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/ApplicationAuditListener.java

```diff
@@ -79,9 +79,12 @@ protected Application prepareApplicationAudit(DelegateExecution execution) {
         if (authentication instanceof JwtAuthenticationToken) {
             submittedBy = ((JwtAuthenticationToken) authentication).getToken().getClaimAsString("preferred_username");
             if(submittedBy.startsWith("service-account")){
-                submittedBy = "Anonymous-user";
+                submittedBy = ANONYMOUS_USER;
```

Comment:
write the conditions in this way.
if (authentication != null) {
  if(authentication instanstance of JwtAutenticationToken authToken){
    submittedBy = authToken.getToken()...
} else if (....)
} else {
submittedBy = SERVICE_ACCOUNT
}

Reply:
done